test/unit/foreman/renderer/snapshots/ProvisioningTemplate/finish/Windows_default_finish.windows10_dhcp.snap.txt

```diff
@@ -0,0 +1,78 @@
+
+
+@echo off
+echo Activating administrator
+net user administrator /active:yes
+
+set ctr=0
+set nettimeout=10
+
+@echo off
+setlocal enableDelayedExpansion
+
+for /f "delims=" %%a in ('ipconfig /all') do (
+    set line=%%a
+    if not "!line:~0,1!"==" " if not "!line:adapter=!"=="!line!" (
+        set name=!line:*adapter =!
+        set name=!name::=!
+    )
+
+    for /f "tokens=1,2,*" %%b in ("%%a") do (
+        if "%%b %%c"=="Physical Address." (
+            set mac=%%d
+            set mac=!mac:*: =!
+            set mac=!mac:-=:!
+            call :tolower mac
+
+        )
+    )
+)
+call :done
+
+:tolower
+for %%L IN (a b c d e f g h i j k l m n o p q r s t u v w x y z) DO SET %1=!%1:%%L=%%L!
+goto :EOF
+
+:done
+
+
+
+echo Syncing time
+w32tm /resync
+w32tm /resync
+
+
+
+
+powershell /c "Get-NetConnectionProfile -InterfaceAlias \"Ethernet0\" | Set-NetConnectionProfile -NetworkCategory Private"
+
+
+
+
+echo Downloading Puppet installer
+wget "" -O C:\puppet-agent-x64-latest.msi
+echo Installing Puppet
+start /w "" msiexec /qn /i C:\puppet-agent-x64-latest.msi PUPPET_AGENT_STARTUP_MODE=Manual PUPPET_SERVER= PUPPET_CA_SERVER= PUPPET_AGENT_ACCOUNT_DOMAIN=snap.example.com PUPPET_AGENT_ACCOUNT_USER=administrator PUPPET_AGENT_ACCOUNT_PASSWORD=""
```

Comment:
What looks funny to me is that `PUPPET_SERVER=` and `PUPPET_CA_SERVER=` are set without any values here. @ekohl Is this okay for the test or is it possible to set default values for host_puppet_server and host_puppet_ca_server?